packages/web/src/pages/chat-page/ChatPage.tsx

```diff
@@ -103,7 +155,14 @@ export const ChatPage = ({ match }: RouteComponentProps<{ id?: string }>) => {
                 className={styles.messageList}
                 chatId={currentChatId}
               />
-              <ChatComposer chatId={currentChatId} />
+              {canSendMessage ? (
+                <ChatComposer chatId={currentChatId} />
+              ) : users.length > 0 ? (
```

Comment:
Feels a bit weird to check `users.length > 0` when there's only one user. Are you building this in anticipation of more users? Otherwise would make sense to set `user = users[0]` somewhere above and just use that.